app/mailers/welcome_mailer.rb

```diff
@@ -0,0 +1,9 @@
+class WelcomeMailer < ApplicationMailer
+
+  def donation_confirmation_email(user, donation)
+    @user = user
```

Comment:
Should we call this 'artist' instead of 'user' for a consistent match to the db schema?